.github/workflows/coverage.yml

```diff
@@ -0,0 +1,69 @@
+name: Coverage
+run-name: Code Coverage
+on:
+  pull_request:
+    branches:
+      - master
+    paths:
+      - '*'
+      - 'package*.json'
```

Comment:
Probably change this before merging to only run `Coverage` when pushed to master? Maybe? Probably.

Reply:
This should be fine actually. On push to a non-master branch should update the coverage.

Reply:
Sure, if there is a better way to manage coverage runs... I mirrored what the testing workflow was working with.

Reply:
Well, now that I think about it, we should collect test coverage whenever the 'test' workflow was run (i.e., on PR to check if a threshold was met).

Reply:
So as long as it's limited to a PR and master then its alright.

Reply:
My suggestion is to run only on pushing to master. This ensures that when `Coverage` is run, it will update according to the code coverage that is in master.

If we run `Coverage` in feature/patch/bug branches, then we are updating the coverage in master without merging. Sometimes PRs can be open for hours/days/etc.

Reply:
If you want the current coverage on the branch, we can print it in the workflow to view it.